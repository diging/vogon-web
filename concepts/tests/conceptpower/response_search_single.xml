<conceptpowerReply xmlns:digitalHPS="http://www.digitalhps.org/">
    <digitalHPS:conceptEntry>
        <digitalHPS:id concept_id="WID-05898568-N-02-program" concept_uri="http://www.digitalhps.org/concepts/WID-05898568-N-02-program">http://www.digitalhps.org/concepts/WID-05898568-N-02-program</digitalHPS:id>
        <digitalHPS:lemma>program</digitalHPS:lemma>
        <digitalHPS:pos>noun</digitalHPS:pos>
        <digitalHPS:description>a series of steps to be carried out or goals to be accomplished; &quot;they drew up a six-step plan&quot;; &quot;they discussed plans for a new bond issue&quot;</digitalHPS:description>
        <digitalHPS:conceptList>WordNet</digitalHPS:conceptList>
        <digitalHPS:creator_id></digitalHPS:creator_id>
        <digitalHPS:equal_to></digitalHPS:equal_to>
        <digitalHPS:modified_by></digitalHPS:modified_by>
        <digitalHPS:similar_to></digitalHPS:similar_to>
        <digitalHPS:synonym_ids>WID-05898568-N-01-plan,WID-05898568-N-03-programme,</digitalHPS:synonym_ids>
        <digitalHPS:type ></digitalHPS:type>
        <digitalHPS:deleted>false</digitalHPS:deleted>
        <digitalHPS:wordnet_id>WID-05898568-N-02-program</digitalHPS:wordnet_id>
        <digitalHPS:alternativeIds>
            <digitalHPS:id concept_id="WID-05898568-N-02-program" concept_uri="http://www.digitalhps.org/concepts/WID-05898568-N-02-program">http://www.digitalhps.org/concepts/WID-05898568-N-02-program</digitalHPS:id>
        </digitalHPS:alternativeIds>
    </digitalHPS:conceptEntry>
</conceptpowerReply>