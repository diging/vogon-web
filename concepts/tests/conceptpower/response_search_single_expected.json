[
  {
    "name": "program",
    "pos": "noun",
    "identities": null,
    "concept_type": null,
    "description": "a series of steps to be carried out or goals to be accomplished; \"they drew up a six-step plan\"; \"they discussed plans for a new bond issue\"",
    "local_identifier": "http://www.digitalhps.org/concepts/WID-05898568-N-02-program",
    "identifier": "http://www.digitalhps.org/concepts/WID-05898568-N-02-program"
  }
]